src/components/d2l-teacher-course-creation-admin.js

```diff
@@ -274,7 +273,7 @@ class TeacherCourseCreationAdmin extends BaseMixin(LitElement) {
 			return html`
 				<d2l-tcc-error
 				.errorMessage="${this.localize('adminPermissionsError')}"
-				.renderBack="${false}">
+				renderBack>
```

Comment:
Did you mean to have `hideBack`?
```suggestion
				hideBack>
```

Reply:
yeah... fixed